scripts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+toml
```

Comment:
why are we using a requirements file? We're already using uv for dependency management